recipes/dotnet_agent.rb

```diff
@@ -68,6 +68,7 @@
     :controller_accesskey => controller['accesskey'],
     :proxy_host => proxy['host'],
     :proxy_port => proxy['port'],
+	:standalone_apps => agent['standalone_apps']
```

Comment:
Are you perhaps mixing tabs and spaces?

On Friday, July 10, 2015, thiru-chidambaram <notifications@github.com>
wrote:

> In recipes/dotnet_agent.rb
> <https://github.com/Appdynamics/appdynamics-cookbooks/pull/18#discussion_r34402059>
> :
>
> > @@ -68,6 +68,7 @@
> >      :controller_accesskey => controller['accesskey'],
> >      :proxy_host => proxy['host'],
> >      :proxy_port => proxy['port'],
> > +	:standalone_apps => agent['standalone_apps']
>
> I'm not sure why the indentations are getting messed up. This happens only
> when I publish it to github. My local files looks okay.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Appdynamics/appdynamics-cookbooks/pull/18/files#r34402059>
> .
>